wiki/Guides/Livestreaming_osu!/en.md

```diff
@@ -0,0 +1,73 @@
+# Livestreaming osu!
+
+This guide will show you how to livestream osu! on Twitch using [OBS Studio](https://obsproject.com/ "Open Broadcaster Software"). Doing so is a relatively simple, although it might require a reliable computer able to both run and livestream osu!.
```

Comment:
```suggestion
This guide will show you how to livestream osu! on Twitch using [OBS Studio](https://obsproject.com/ "Open Broadcaster Software"). Doing so is relatively simple, although it might require a reliable computer able to both run and livestream osu!.
```
my mistake here, whoops